example/main.tf

```diff
@@ -2,14 +2,19 @@ module "cloudfront_auth" {
   source = "../"
 
   auth_vendor             = "cognito"
+  authz                   = "2"  # Is this a number or a name?
   cloudfront_distribution = "private.example.com"
+  cloudfront_aliases      = ["secret.example.com"]
+  base_uri                = "https://auth-xxxxxx.auth.us-east-1.amazoncognito.com"
   client_id               = "CHANGE_ME"
   client_secret           = "CHANGE_ME"
+  nodejs_version          = "12.18.0"
   redirect_uri            = "https://private.example.com/callback"
-  github_organization     = "exampleorg"
 
   bucket_name                    = "private.example.com"
   region                         = "eu-west-1"
+  # TODO: probably wrong
+  bucket_access_roles_arn_list   = []
```

Comment:
"A Role ARN which granted RW rights to bucket (to be used by instance profiles in order to upload/cleanup bucket content)"